client/src/modules/stock/inventories/registry.js

```diff
@@ -228,6 +236,8 @@ function StockInventoriesController(
   }
 
   function startup() {
+    setDefaultFilters();
```

Comment:
How do other registries set their default filters?  I think they do it in a service.  Can you move this logic to the service?

Reply:
If we set it in the stock service, it will be used everywhere (Depot Management, Requisition, Items in stock, Stock Assignment, Stock Movement, Stock Lots) since we don't need it everywhere so i restrict it to this module only.